src/metadata-stream.js

```diff
@@ -6,12 +6,24 @@
 (function(window, videojs, undefined) {
   'use strict';
   var
```

Comment:
Well I see why `var` is there, I'm more asking about the benefits of having an 80 line multiple var statement. Not trying to be contrary, I've just seen this pattern pop up in a few different projects recently.